tests/nlu/classifiers/test_fallback_classifier.py

```diff
@@ -85,7 +85,7 @@ def test_predict_fallback_intent(message: Message, component_config: Dict):
     classifier = FallbackClassifier(component_config=component_config)
     classifier.process(message)
 
-    confidence = 1 - old_message_state.data[INTENT][PREDICTED_CONFIDENCE_KEY]
```

Comment:
btw this was a bad practice. I'd classify this as [TTDD (Tautological TDD)](https://www.thoughtworks.com/de/insights/blog/mockists-are-dead-long-live-classicists) since it does the same calculation as the component itself

Reply:
ah right TIL 😄 Should I change L88 to `confidence = component_config[THRESHOLD_KEY]` or is the current change all good?

Reply:
Yes, that would be great 👍🏻 Ideally copy the dictionary before (or assign the variable before initializing the classifier) to make sure `FallbackClassifier` doesn't alter the value somehow